changelogs/unreleased/add-xxhash-to-digest-module.md

```diff
@@ -0,0 +1,3 @@
+## feature/lua/digest
+
+ * Introduce new hash types in digest module - `xxhash32` and `xxhash64`.
```

Comment:
You should also add the issue link. See the other changelog entries for examples.

Reply:
Fixed